### PR TITLE
Fix Bug 1126583: Create Fellowship single page site.

### PR DESCRIPTION
**Pull request by stephaniehobson**
Added URL, HTML, CSS, JS, and image asset to support creation of /fellowship page.

Moved details/summary shim into wiki.js for wider use.

Test with waffle flag "fellowship_applications_open" both on and off.

### CODE REVIEW

File: media/redesign/js/fellowship.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+(function (win, $) {
+    'use strict';
+
+    (function() {
```
Comment (darkwing) [inline media/redesign/js/fellowship.js]: Don't need this extra closure if this file only does one thing.

**Comment by darkwing** [comment]
Outside of this, everything works and looks good.

**Comment by stephaniehobson** [comment]
Updated! Also added links around github and twitter profiles.

**Comment by darkwing** [comment]
Awesome, checking again.

File: media/redesign/stylus/fellowship.styl [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+@require 'includes/mixins';
+
+@require 'components/fellowship/base';
+@require 'components/fellowship/details-list';
+@require 'components/fellowship/pullquote';
+@require 'components/fellowship/hero';
+@require 'components/fellowship/cta';
+
+
+
```
Comment (darkwing) [inline media/redesign/stylus/fellowship.styl]: Lamest nit ever but empty lines.

**Comment by darkwing** [comment]
So everything here works but what I'd recommend is that we use `target="blank"` on the social links.  Maybe use a small popup..like http://davidwalsh.name   :D

**Comment by stephaniehobson** [comment]
Changes made!

**Comment by darkwing** [comment]
This keeps failing but I can't see why... :/

**Comment by stephaniehobson** [comment]
Augh, I got settings.py caught up in one of my commits. I'll fix it.

**Comment by groovecoder** [comment]
Bah, the latest failure is something that snuck thru on [my PR to wrap user registration in a waffle flag](https://github.com/mozilla/kuma/pull/3091). :frowning: I sent https://github.com/mozilla/kuma/pull/3092 to fix it, so it's okay to merge this.

**Pull request merged**